master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -698,88 +698,97 @@ private[celeborn] class Master(
 
   override def getWorkerInfo: String = {
     val sb = new StringBuilder
+    sb.append("====================== Workers Info in Master ===========================")
     workersSnapShot.asScala.foreach { w =>
-      sb.append("==========WorkerInfos in Master==========\n")
+      sb.append(s"${w.toUniqueId().padTo(50, " ")}in Master\n")
       sb.append(w).append("\n")
 
+      sb.append("\n")
       val workerInfo = requestGetWorkerInfos(w.endpoint)
         .workerInfos.asJava
         .get(0)
 
-      sb.append("==========WorkerInfos in Workers==========\n")
+      sb.append(s"${w.toUniqueId().padTo(50, " ")}in Worker\n")
       sb.append(workerInfo).append("\n")
+      sb.append("\n")
 
       if (w.hasSameInfoWith(workerInfo)) {
-        sb.append("Consist!").append("\n")
+        sb.append(s"${w.toUniqueId().padTo(50, " ")}status consist!\n")
       } else {
-        sb.append("[ERROR] Inconsistent!").append("\n")
+        sb.append(s"${w.toUniqueId().padTo(50, " ")}status not consist!\n")
       }
+      sb.append("======================================================================")
     }
 
     sb.toString()
   }
 
   override def getLostWorkers: String = {
     val sb = new StringBuilder
-    sb.append("========== Lost WorkerInfos in Master==========\n")
-    lostWorkersSnapshot.asScala.map { case (worker, time) =>
-      sb.append(s"${worker.toUniqueId()}      $time\n")
+    sb.append("======================= Lost Workers in Master ========================\n")
+    lostWorkersSnapshot.asScala.foreach { case (worker, time) =>
+      sb.append(s"${worker.toUniqueId().padTo(50, " ")}$time\n")
     }
     sb.toString()
   }
 
   override def getBlacklistedWorkers: String = {
     val sb = new StringBuilder
-    sb.append("==========Blacklisted WorkerInfos in Master==========\n")
-    statusSystem.blacklist.asScala.map { worker =>
+    sb.append("==================== Blacklisted Workers in Master =====================\n")
+    statusSystem.blacklist.asScala.foreach { worker =>
       sb.append(s"${worker.toUniqueId()}\n")
     }
     sb.toString()
   }
 
   override def getThreadDump: String = {
     val sb = new StringBuilder
-    val threadDump = Utils.getThreadDump()
-    sb.append("==========Master ThreadDump==========\n")
-    sb.append(threadDump).append("\n")
-    workersSnapShot.asScala.foreach(w => {
```

Comment:
So we don't return workers' threaddump anymore, do you think we need a single request to return all workers' threaddump?

Reply:
> So we don't return workers' threaddump anymore, do you think we need a single request to return all workers' threaddump?

The message is so noisy and I don't think we need to return all messages together, even if we want to build a machine admin website,   we always open one machine's stack... won't open it together...

Reply:
reasonable to me